_maps/_basemap.dm

```diff
@@ -15,6 +15,7 @@
 		#include "map_files\HelioStation\HelioStation.dmm"
 		#include "map_files\SeleneStation\SeleneStation.dmm"
 		#include "map_files\tramstation\tramstation.dmm"
+		#include "map_files\tramstation\solitairestation.dmm"
```

Comment:
tramstation\solitairestation

Reply:
S should be capitalized too

Reply:
also should probably go right below selene rather than below tram